error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/AssertJIsNull.java

```diff
@@ -19,28 +21,36 @@
 import com.google.errorprone.fixes.SuggestedFixes;
 import com.google.errorprone.matchers.Description;
 import com.google.errorprone.matchers.Matcher;
+import com.google.errorprone.util.ASTHelpers;
 import com.sun.source.tree.MethodInvocationTree;
 
 /**
- * A {@link BugChecker} that flags AssertJ {@code isEqualTo(null)} checks for simplification.
+ * A {@link BugChecker} that flags AssertJ {@code isEqualTo(null)}, {@code isSameAs(null)}, {@code
+ * isNotEqualTo(null)} and {@code isNotSameAs(null)} checks for simplification.
  *
  * <p>This bug checker cannot be replaced with a simple Refaster rule, as the Refaster approach
- * would require that all overloads of {@link org.assertj.core.api.Assert#isEqualTo(Object)} (such
- * as {@link org.assertj.core.api.AbstractStringAssert#isEqualTo(String)}) are explicitly
- * enumerated. This bug checker generically matches all such current and future overloads.
+ * would require that all overloads of the mentioned methods (such as {@link
+ * org.assertj.core.api.AbstractStringAssert#isEqualTo(String)}) are explicitly enumerated. This bug
+ * checker generically matches all such current and future overloads.
  */
 @AutoService(BugChecker.class)
 @BugPattern(
-    summary = "Prefer `.isNull()` over `.isEqualTo(null)`",
+    summary = "Prefer `.isNull()` and `.isNotNull()` over more verbose alternatives",
     link = BUG_PATTERNS_BASE_URL + "AssertJIsNull",
     linkType = CUSTOM,
     severity = SUGGESTION,
     tags = SIMPLIFICATION)
 public final class AssertJIsNull extends BugChecker implements MethodInvocationTreeMatcher {
```

Comment:
I think that the check's name is fine to keep as-is, but open to alternatives. (If we do rename, then the PR title and suggested commit message must be updated accordingly.)

Reply:
I think that makes sense yes! I like the name.

Reply:
K, will change.